pick/__init__.py

```diff
@@ -60,13 +77,20 @@ def get_title_lines(self):
 
     def get_option_lines(self):
         lines = []
-
         for index, option in enumerate(self.options):
             if index == self.index:
                 prefix = self.indicator
             else:
                 prefix = len(self.indicator) * ' '
-            line = '{0} {1}'.format(prefix, option)
+
+            if self.multi_select:
+                if index in self.all_selected:
+                    format = curses.color_pair(1)
```

Comment:
this could be written as:

```
if self.multi_select and index in self.all_selected:
    format = curses.color_pair(1)
    line = ('{0} {1}'.format(prefix, option), format)
else:
    line = '{0} {1}'.format(prefix, option)
```

right?